zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/ElasticsearchClientFactory.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter;
+
+import co.elastic.clients.elasticsearch.ElasticsearchClient;
+import co.elastic.clients.json.jackson.JacksonJsonpMapper;
+import co.elastic.clients.transport.ElasticsearchTransport;
+import co.elastic.clients.transport.rest_client.RestClientTransport;
+import io.camunda.exporter.NoopExporterConfiguration.ElasticsearchConfig;
+import io.camunda.exporter.schema.ElasticsearchEngineClient;
+import org.apache.http.HttpHost;
+import org.elasticsearch.client.RestClient;
+
+public final class ElasticsearchClientFactory {
```

Comment:
is this being used anywhere in the scope of this PR?

Reply:
my bad this was a WIP I forgot to delete